app/controllers/v0/user_preferences_controller.rb

```diff
@@ -29,5 +33,9 @@ def user_preference_params
     def requested_user_preferences
       user_preference_params['_json']
     end
+
+    def user_preferences
+      @user_preferences ||= UserPreference.all_preferences_with_choices(current_user.account_id)
```

Comment:
Are we able to call `account_id` off of a `User`?  I believe this would be `account.id`, yes?